### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by offerijns**
Add pallet descriptions and description of Centrifuge to README, separate development instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,223 +1,56 @@
...
+    <br />
+    <a href="https://docs.centrifuge.io/build/cent-chain/"><strong>Read the documentation »</strong></a>
+  </p>
+</p>
 
-The Centrifuge Chain is [Centrifuge](https://centrifuge.io)'s [Substrate](https://github.com/paritytech/substrate)-based chain.
+## About Centrifuge
+Centrifuge is the infrastructure that facilitates the decentralized financing of real-world assets natively on-chain, creating a fully transparent market which allows borrowers and lenders to transact without unnecessary intermediaries. Asset pools are fully collateralized, liquidity providers have legal recourse, and the protocol is asset-class agnostic with pools for assets spanning mortgages, invoices, microlending and consumer finance. Ultimately, the protocol aims to lower the cost of borrowing for businesses around the world, while providing DeFi users with a stable source of collateralized yield that is uncorrelated to the volatile crypto markets. By bringing the entire structured credit market on-chain across securitization, tokenization, privacy, governance, and liquidity integrations, Centrifuge is building a more transparent, affordable, and limitless financial system.
```
Comment (NunoAlexandre) [inline README.md:15]: 🥲 ❤️

File: README.md [hunk 2]
```diff
@@ -1,223 +1,56 @@
...
-3. Onboard the parachain
-    This step will have the targeted parachain onboarded in the relay chain. The parachain will NOT produce blocks until this step is completed successfully.
-    ```bash
-    ./scripts/init.sh onboard-parachain
-    ```
-    When you have run the command, you should see in the relay-chain dashboard that there is a parachain
-    that will be onboarded in one/two minutes.
-    Once onboarded, block production should start soon after in the parachain.
+## Developing
+Instructions for building, testing and developing Centrifuge Chain can be found in [`docs/DEVELOPING.md`](docs/DEVELOPING.md).
```
Comment (NunoAlexandre) [inline README.md:53]: `nit`

```suggestion
Instructions for building, testing, and developing Centrifuge Chain can be found in [`docs/DEVELOPING.md`](docs/DEVELOPING.md).
```

[Source](https://www.grammar-monster.com/lessons/commas_the_Oxford_comma.htm)

**Review by NunoAlexandre — COMMENTED** [review]
Really nice @offerijns 😃

File: README.md [hunk 3]
```diff
@@ -1,223 +1,56 @@
...
+  </p>
+</p>
 
-The Centrifuge Chain is [Centrifuge](https://centrifuge.io)'s [Substrate](https://github.com/paritytech/substrate)-based chain.
+## About Centrifuge
+Centrifuge is the infrastructure that facilitates the decentralized financing of real-world assets natively on-chain, creating a fully transparent market which allows borrowers and lenders to transact without unnecessary intermediaries. Asset pools are fully collateralized, liquidity providers have legal recourse, and the protocol is asset-class agnostic with pools for assets spanning mortgages, invoices, microlending and consumer finance. Ultimately, the protocol aims to lower the cost of borrowing for businesses around the world, while providing DeFi users with a stable source of collateralized yield that is uncorrelated to the volatile crypto markets. By bringing the entire structured credit market on-chain across securitization, tokenization, privacy, governance, and liquidity integrations, Centrifuge is building a more transparent, affordable, and limitless financial system.
 
-## Build
+## Building blocks
+The Substrate runtime makes use of various custom pallets that are found in the crates folder.
```
[the remark this answers is not in the document]
Comment (gruberb) [reply README.md:18]: Maybe this one would be better?

On top of the [Substrate FRAME](https://docs.substrate.io/reference/frame-pallets/) framework, we built our own custom pallets which can be found inside the `pallets` folder. The following list gives you a brief overview, and links to the corresponding documentation.

File: README.md [hunk 4]
```diff
@@ -1,223 +1,56 @@
-# centrifuge-chain
+<p align="center">
+  <a href="https://github.com/centrifuge/centrifuge-chain">
+    <img alt="Centrifuge" src="/docs/images/banner.png">
```
[the remark this answers is not in the document]
Comment (offerijns) [reply README.md]: I hadn't considered this at all, great point! Looks like SVGs work, I just replaced it with one (9.1kb!).

File: docs/DEVELOPING.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+Great! You have already compile the Centrifuge Chain!
+
+## Tests
+
+There are two kinds of tests, one related to how the *Centrifuge Chain* works itself
+and another one to verify how it works in a more real environment as a parachain.
+
+### Chain tests
+
+The following command will the unit and the integration tests:
```
Comment (mustermeiszer) [inline docs/DEVELOPING.md:28]: *run

**Review by mustermeiszer — DISMISSED** [review]
Just one typo. That looks really clean and nice. Also, the development section, whoever updated this, is greatly structured and understandable!

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (NunoAlexandre) [reply README.md]: Super nice :D thanks!

**Review by NunoAlexandre — APPROVED** [review]

**Review by gruberb — APPROVED** [review]
🎉🚀

**Pull request merged**